grails-app/conf/BuildConfig.groovy

```diff
@@ -102,6 +102,10 @@ grails.project.dependency.resolution = {
             excludes "xercesImpl", "groovy", "commons-lang", "commons-codec"
         }
 
+        // REST client
+        compile 'org.apache.httpcomponents:httpclient:4.5.12'
```

Comment:
Out of curiosity, why this was added now? Wasnt this required ever before?

Reply:
It might be an implicit requirement of some other dependency but I don't think I've ever used the library within our code base. In any case I need a compile-time dependency so that I could write code using the HttpClient and related classes.